### PR TITLE
Gist comments

### PR DESCRIPTION
**Pull request by kayladnls**

### CODE REVIEW

File: lib/Github/Api/Gists.php [hunk 1]
```diff
@@ -73,4 +73,17 @@ public function unstar($id)
     {
         return $this->delete('gists/'.rawurlencode($id).'/star');
     }
+
+    /**
+     * List a gists comments.
```
Comment (stof) [inline lib/Github/Api/Gists.php]: this is not listing comments

File: lib/Github/Api/Gists.php [hunk 2]
```diff
@@ -73,4 +73,17 @@ public function unstar($id)
...
+     *
+     * @link http://developer.github.com/v3/gists/comments/
+     *
+     * @return Comments
+     */
+    public function comments()
+    {
+        return new Comments($this->client);
+    }
+
```
Comment (stof) [inline lib/Github/Api/Gists.php]: you have an extra empty line here

File: test/Github/Tests/Api/Gist/CommentsTest.php [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+
+namespace Github\Tests\Api\Gist;
+
+use Github\Tests\Api\TestCase;
+
+class CommentsTest extends TestCase
+{
+    /**
+     * @test
+     */
```
Comment (stof) [inline test/Github/Tests/Api/Gist/CommentsTest.php]: the phpdoc should be just above the method it documents

File: lib/Github/Api/Gist/Comments.php [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+/**
+ *
+ * @link   https://developer.github.com/v3/gists/comments/
+ * @author Kayla Daniels <kayladnls@gmail.com>
+ */
+class Comments extends AbstractApi
+{
+    public function all($gist)
+    {
+        return $this->get('gists/'.rawurlencode($gist)."/comments");
```
Comment (pilot) [inline lib/Github/Api/Gist/Comments.php]: use single quotes, check below

**Comment by kayladnls** [comment]
Any word on this? I could really use the features for an upcoming project.

File: lib/Github/Api/Gist/Comments.php [hunk 5]
```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace Github\Api\Gist;
+
+use Github\Api\AbstractApi;
+
+/**
+ *
```
Comment (pilot) [inline lib/Github/Api/Gist/Comments.php]: rm line

File: lib/Github/Api/Gist/Comments.php [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+    }
+
+    public function create($gist, $body)
+    {
+        return $this->post('gists/'.rawurlencode($gist).'/comments', array($body));
+    }
+
+    public function update($gist, $comment_id, $body)
+    {
+        return $this->patch('gists/'.rawurlencode($gist)."/comments/".rawurlencode($comment_id), array($body));
```
Comment (pilot) [inline lib/Github/Api/Gist/Comments.php]: single quotes

File: lib/Github/Api/Gist/Comments.php [hunk 7]
```diff
@@ -0,0 +1,38 @@
...
+    }
+
+    public function remove($gist, $comment)
+    {
+        return $this->delete('gists/'.rawurlencode($gist)."/comments/".rawurlencode($comment));
```
Comment (pilot) [inline lib/Github/Api/Gist/Comments.php]: single quotes

File: test/Github/Tests/Api/Gist/CommentsTest.php [hunk 8]
```diff
@@ -0,0 +1,96 @@
...
+        $api = $this->getApiMock();
+        $api->expects($this->once())
+            ->method('get')
+            ->with('gists/123/comments/123')
+            ->will($this->returnValue($expectedValue));
+
+        $this->assertEquals($expectedValue, $api->show(123, 123));
+    }
+
+
```
Comment (pilot) [inline test/Github/Tests/Api/Gist/CommentsTest.php]: rm extra spaces, check under the below methods too

File: test/Github/Tests/Api/Gist/CommentsTest.php [hunk 9]
```diff
@@ -0,0 +1,96 @@
...
+     * @test
+     */
+    public function shouldCreateGistComment()
+    {
+        $expectedValue = array('comment1data');
+
+        $api = $this->getApiMock();
+        $api->expects($this->once())
+            ->method('post')
+            ->with('gists/123/comments', array("Test body"))
```
Comment (pilot) [inline test/Github/Tests/Api/Gist/CommentsTest.php]: check that you consistence in case to use single quotes everywhere

**Comment by pilot** [comment]
@kayladnls thanks!

**Comment by pilot** [comment]
https://github.com/KnpLabs/php-github-api/releases/tag/1.4.6

**Pull request merged**